ivadomed/main.py

```diff
@@ -671,12 +671,17 @@ def cmd_test(context):
         rdict['gt'] = preds.cpu()
         batch.update(rdict)
 
+        if batch["input"].shape[0] > 1:
```

Comment:
It seems that we are entering in this if condition when we are in monochannel --> will look at it

Reply:
oups yes clearly because of the batch size should be shape[1]